### PR TITLE
multiprocessing: shutdown_timeout param to BaseManager

### PR DESCRIPTION
**Pull request by JelleZijlstra**
python/cpython#32112

### CODE REVIEW

File: stdlib/multiprocessing/managers.pyi [hunk 1]
```diff
@@ -76,9 +76,21 @@ class Server:
...
+    else:
+        def __init__(
+            self,
+            address: Any | None = ...,
+            authkey: bytes | None = ...,
+            serializer: str = ...,
+            ctx: BaseContext | None = ...,
+            *,
+            shutdown_timeout: float = ...,
+        ) -> None: ...
```
Comment (AlexWaygood) [inline stdlib/multiprocessing/managers.pyi:92]: Shouldn't these be the other way around? 🤔

Comment (JelleZijlstra) [reply stdlib/multiprocessing/managers.pyi]: Indeed, fixed. I think that's a sign I've done enough of these for today.

Comment (AlexWaygood) [reply stdlib/multiprocessing/managers.pyi]: Are you using stubtest to find these, out of interest? :)

Comment (JelleZijlstra) [reply stdlib/multiprocessing/managers.pyi]: No, just going over `git log` in CPython

**Review by AlexWaygood — APPROVED** [review]

**Pull request merged**